briefcase/app.py

```diff
@@ -262,6 +266,75 @@ def install_code(self):
                 '.'
             ])
 
+    @property
+    def launcher_header(self):
+        """
+        Override the shebang line for launcher scripts
+        This should return a suitable relative path which will find the
+        bundled python for the relevant platform
+        """
+        return None
```

Comment:
Should this be a `return None`, or a `NotImplementedException`?

Reply:
Actually no, `None` means the setuptools default will be used, which may be suitable depending on the platform.
I'll update the comment to make this more clean.